docs/configuration/configuration.md

```diff
@@ -1546,6 +1550,79 @@ for a detailed example of configuring Prometheus for Kubernetes.
 You may wish to check out the 3rd party [Prometheus Operator](https://github.com/coreos/prometheus-operator),
 which automates the Prometheus setup on top of Kubernetes.
 
+### `<kuma_sd_config>`
+
+Kuma SD configurations allow retrieving scrape target from the [Kuma](https://kuma.io) control plane.
+
+This SD discovers "monitoring assignments" based on Kuma [Dataplane Proxies](https://kuma.io/docs/latest/documentation/dps-and-data-model),
+via the MADS v1 (Monitoring Assignment Discovery Service) xDS API, and will create a target for each proxy
+inside a Prometheus-enabled mesh.
+
+Available Kuma-specific meta labels:
+
+* `__meta_kuma_mesh`: the name of the proxy's Mesh 
+* `__meta_kuma_dataplane`: the name of the proxy
+* `__meta_kuma_service`: the name of the proxy's associated Service
+* `__meta_kuma_label_<tagname>`: each tag of the proxy
+
+See below for the configuration options for Kuma MonitoringAssignment discovery:
+
+```yaml
+# Address of the Kuma Control Plane's MADS xDS server.
+server: <string>
+# An arbitrary identifier to send to the MADS server, which should be unique to this instance.
```

Comment:
Is this really a strict requirement or could we hide it from the user? I would think that because we have the nonce it does not really have to be unique?

Reply:
From my understamding we could remove this parameter for now and not expose it to the user.

Reply:
Sounds good – we'll default to just `"prometheus"` if the FQDN cannot be found for some reason. This doesn't currently matter for the Kuma case, but could affect how the xDS server chooses to partition its resource cache.